trunk/SUAVE/Components/Energy/Networks/Battery_Propeller.py

```diff
@@ -155,26 +171,34 @@ def evaluate_thrust(self,state):
         battery.energy_calc(numerics)        
     
         # Pack the conditions for outputs
+        a                    = conditions.freestream.speed_of_sound
+        R                    = propeller.tip_radius
         rpm                  = motor.outputs.omega*60./(2.*np.pi)
         current              = esc.outputs.currentin
         battery_draw         = battery.inputs.power_in 
         battery_energy       = battery.current_energy
         voltage_open_circuit = battery.voltage_open_circuit
         voltage_under_load   = battery.voltage_under_load    
           
-        conditions.propulsion.rpm                  = rpm
-        conditions.propulsion.current              = current
-        conditions.propulsion.battery_draw         = battery_draw
-        conditions.propulsion.battery_energy       = battery_energy
-        conditions.propulsion.voltage_open_circuit = voltage_open_circuit
-        conditions.propulsion.voltage_under_load   = voltage_under_load  
-        conditions.propulsion.motor_torque         = motor.outputs.torque
-        conditions.propulsion.propeller_torque     = Q
+        conditions.propulsion.rpm                   = rpm
+        conditions.propulsion.current               = current
+        conditions.propulsion.battery_draw          = battery_draw
+        conditions.propulsion.battery_energy        = battery_energy
+        conditions.propulsion.voltage_open_circuit  = voltage_open_circuit
+        conditions.propulsion.voltage_under_load    = voltage_under_load  
+        conditions.propulsion.motor_torque          = motor.outputs.torque
+        conditions.propulsion.propeller_torque      = Q
+        conditions.propulsion.battery_specfic_power = -(battery_draw/1000)/battery.mass_properties.mass 
```

Comment:
What is this hard coded 1000?

Reply:
Conversion to kW. I added a comment

Reply:
On second thought, to keep standard units, I think we should not divide by 1000. I will correct all of the associated files